aten/src/ATen/native/Scalar.cpp

```diff
@@ -11,11 +11,17 @@
 #include <ATen/ops/item_native.h>
 #endif
 
+#include <torch/csrc/autograd/grad_mode.h>
+
 namespace at::native {
 
 Scalar item(const Tensor& self) {
   auto numel = self.sym_numel();
   TORCH_CHECK(numel == 1, "a Tensor with ", numel, " elements cannot be converted to Scalar");
+  if (numel == 1 && torch::autograd::GradMode::is_enabled() && self.requires_grad()) {
```

Comment:
numel == 1 isn't needed here I think since it's being checked on the above line

Reply:
Good catch. I've made the change.